gulpfile.js

```diff
@@ -85,7 +85,7 @@ gulp.task('vendor', function vendorTask(done) {
 
 // based on https://github.com/gulpjs/gulp/blob/master/docs/recipes/browserify-with-globs.md,
 // except we use the Promise returned by globby, instead of passing it a callback
-gulp.task('scripts', ['lint'], function scriptsTask() {
+gulp.task('scripts', function scriptsTask() {
```

Comment:
@lmorchard @pdehaan Could we maybe have an option for linting on watch? It kinda drives me crazy personally to have my build failing over forgotten semicolons, extra space, and some of the other nitpicky rules. But I'm always happy to have them fixed up before pushing.